gensim/test/test_utils.py

```diff
@@ -97,6 +97,10 @@ def test_check_output(self):
         res = utils.check_output(args=["echo", "hello"])
         self.assertEqual(res, b'hello\n')
 
+    def test_check_output_exception(self):
+        error = utils.check_output(args=['ldfs'])
+        self.assertEqual(error, "subprocess.check_output could not execute command ' ldfs '")
```

Comment:
```test_utils.pyt``` imports ```utils.py``` from the installed ```gensim```. So this test case is still running the old version of ```check_output``` and therefore the exception test fails. However, if you change the import to use the current version of ```utils.check_output```, all tests pass.